CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [v1.6.0] - 2018-06-06
```

Comment:
Should we include or link to the upgrade guide here?

Reply:
Yes! I guess I should wrap that up.